scripts/release.sh

```diff
@@ -2,51 +2,12 @@
 
 set -e
 
-GOLANG_VERSION="$1"
-GOTENBERG_VERSION="$2"
-GOTENBERG_USER_GID="$3"
-GOTENBERG_USER_UID="$4"
-NOTO_COLOR_EMOJI_VERSION="$5"
-PDFTK_VERSION="$6"
-DOCKER_REPOSITORY="$7"
-
-if [ "$GOTENBERG_VERSION" == "edge" ]; then
-  docker buildx build \
-    --build-arg GOLANG_VERSION="$GOLANG_VERSION" \
-    --build-arg GOTENBERG_VERSION="$GOTENBERG_VERSION" \
-    --build-arg GOTENBERG_USER_GID="$GOTENBERG_USER_GID" \
-    --build-arg GOTENBERG_USER_UID="$GOTENBERG_USER_UID" \
-    --build-arg NOTO_COLOR_EMOJI_VERSION="$NOTO_COLOR_EMOJI_VERSION" \
-    --build-arg PDFTK_VERSION="$PDFTK_VERSION" \
-    --platform linux/amd64 \
-    --platform linux/arm64 \
-    --platform linux/386 \
-    --platform linux/arm/v7 \
-    -t "$DOCKER_REPOSITORY/gotenberg:edge" \
-    --push \
-    -f build/Dockerfile .
-
-  # Cloud Run variant.
-  # Only linux/amd64! See https://github.com/gotenberg/gotenberg/issues/505#issuecomment-1264679278.
-  docker buildx build \
-    --build-arg DOCKER_REPOSITORY="$DOCKER_REPOSITORY" \
-    --build-arg GOTENBERG_VERSION="$GOTENBERG_VERSION" \
-    --platform linux/amd64 \
-    -t "$DOCKER_REPOSITORY/gotenberg:edge-cloudrun" \
-    --push \
-    -f build/Dockerfile.cloudrun .
-
-  exit 0
-fi
-
-GOTENBERG_VERSION="${GOTENBERG_VERSION//v}"
-IFS='.' read -ra SEMVER <<< "$GOTENBERG_VERSION"
-VERSION_LENGTH=${#SEMVER[@]}
-
-if [ "$VERSION_LENGTH" -ne 3 ]; then
-  echo "$VERSION is not semver."
-  exit 1
-fi
+GOLANG_VERSION="1.22.5"
+GOTENBERG_VERSION="8.7.0"
+GOTENBERG_USER_GID="1001"
+GOTENBERG_USER_UID="1001"
+NOTO_COLOR_EMOJI_VERSION="15.1"
+PDFTK_VERSION="v3.3.3"
```

Comment:
How did these versions get picked? How will we know how to update these over time? Also, might be good to quickly add the build instructions to the README unless it's as simple as just calling this script with no arguments or anything

Reply:
You just run the script, but I'll add instructions to update the tag

Reply:
And those were just the latest versions available. Unfortunately I couldn't find any centralized source of truth for what these versions should be as it seems like Gotenberg just takes care of that under the hood when they push their releases to docker hub. I'll add links so that whenever the image gets updated we can quickly look for updates for each dependency